### PR TITLE
Print git branch name and commit Id in LISAv2 log

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
Add printing git branch and commit number if available

### CODE REVIEW

File: LISAv2-Framework.psm1 [hunk 1]
```diff
@@ -164,6 +181,11 @@ function Start-LISAv2 {
 			$plainTextSummary = $testController.TestSummary.GetPlainTextSummary()
 			Write-LogInfo $plainTextSummary
 
+			if ( $currentBranch ) {
+				Write-LogInfo "Current Branch: $currentBranch"
```
Comment (iamshital) [inline LISAv2-Framework.psm1]: Just a suggestion..

Current Branch --> Current Git Branch :
Current Commit --> Current Git Commit :

File: LISAv2-Framework.psm1 [hunk 2]
```diff
@@ -164,6 +181,11 @@ function Start-LISAv2 {
...
+				Write-LogInfo "Current Commit ID: $CommitId"
+			}
+
```
Comment (iamshital) [inline LISAv2-Framework.psm1]: Too many unnecessary commits. As this is new (single) enhancement, it should be completed in one commit.
(You can always use "`git commit --amend`" to push the changes / corrections in last committed commit.

Comment (juhlee-microsoft) [reply LISAv2-Framework.psm1]: We do swuash and merge. No worry

Comment (juhlee-microsoft) [reply LISAv2-Framework.psm1]: We do swuash and merge. No worry

File: LISAv2-Framework.psm1 (hunk 1 above)
Comment (juhlee-microsoft) [reply LISAv2-Framework.psm1]: Fixed in 92bc9175f923e8e269a3d2f287a62e1f56a78e72

**Pull request merged**